### ISSUE TITLE
UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by Chinouth2144** [earth-insights/ZoomEarth#3]
请问在进行grpo训练时，--gradient_checkpointing设置为True， 会提示UserWarning: None of the inputs have requires_grad=True. Gradients will be None，不知道是否有影响？
如果设置为False，不会报错，但是会OOM。
